index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+          <h1 id="title" class="display-3">Title</h1>
+
+        </div>
+        <p id="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere laudantium quaerat ipsa quidem dolores fugit enim saepe blanditiis id deleniti odit tempora corporis consequuntur libero nobis incidunt, ut neque nisi!</p>
```

Comment:
I kindly suggest to indent this p tag so you can better understand the hierarchy of the HTML elements